src/index.html

```diff
@@ -7,8 +7,57 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin = "anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="link__logo">
+        <img
+          src="./images/logo.png"
+          alt="moyo_logo"
+          class="logo__icon"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">APPLE</a>
```

Comment:
Add corresponding values for href attributes for each link
```suggestion
            <a href="#apple" class="nav__link is-active">APPLE</a>
```

Reply:
I think this comment is not valid, because we don't have blocks with corresponding ids right now

Reply:
@Anna-Tkachenko hello